### DISCUSSION TITLE
How would one add wezterm to macOS dock?

### DISCUSSION THREAD

**Discussion by hasrthur** [wezterm/wezterm#5773] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

When I run `wezterm` from mac terminal I can add the resulting window to dock. But when I try to reopen using the icon in dock it first starts mac terminal which will spawn wezterm.

What is the preferred way of adding wezterm to the dock?

**Comment by hasrthur** [comment]
I must add that I install it using home-manager for nix

**Comment by hasrthur** [comment]
Ok, so the launcher is created, it's just in another place and home manager doesn't yet handle this: https://github.com/nix-community/home-manager/issues/1341

**Reply by feketegy** [reply]
If you open WezTerm doesn't it show up in the Dock? If it does, then you can just drag the icon wherever in the Dock and it will stay there.
